src/widgets/ListNftWidget/helpers/index.ts

```diff
@@ -0,0 +1,33 @@
+import { ListNftState } from '../ListNftWidget';
+
+export const getTitle = (state: ListNftState): string => {
+  if (state === ListNftState.review) {
+    return 'Review Swap';
+  }
+
+  if (state === ListNftState.approve) {
+    return 'Approve in Wallet';
+  }
+
+  if (state === ListNftState.approving) {
+    return 'Transaction in Progress';
+  }
+
+  if (state === ListNftState.sign) {
+    return 'Sign in Wallet';
+  }
+
+  if (state === ListNftState.listing) {
+    return 'Listing in progress';
+  }
+
+  if (state === ListNftState.success) {
+    return 'Successfully listed';
+  }
+
+  if (state === ListNftState.failed) {
+    return 'Listing failed';
+  }
+
+  return 'List NFT';
+};
```

Comment:
I'm guessing you don't like elif's? :stuck_out_tongue_closed_eyes:

Reply:
nope 😛

Reply:
This is nicer to read, no doubt